Modules/MachineLearning/2.0/deploy.json

```diff
@@ -0,0 +1,228 @@
+      "defaultValue": "",
+      "metadata": {
+        "description": "Optional. The business owner for the application"
+      }
+    },
+    "tagOwnerEmail": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Optional. The Email address of the business owner for the application"
+      }
+    }
+  },
+  "variables": {
+    "machinelearningName": "[parameters('workspaceName')]",
+    "machinelearningResourceId": "[resourceId('Microsoft.MachineLearning/workspaces', parameters('workspaceName'))]",
+    "storageAccountResourceId": "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]",
+    "pidName": "[concat('pid-', parameters('cuaId'))]"
+  },
+  "resources": [
+    {
+      "condition": "[not(empty(parameters('cuaId')))]",
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2018-02-01",
+      "name": "[variables('pidName')]",
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "resources": []
+        }
+      }
+    },
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "[providers('Microsoft.Storage','storageAccounts').apiVersions[0]]",
+      "name": "[parameters('storageAccountName')]",
+      "location": "[resourceGroup().location]",
+      "tags": {
+        "Environment": "[parameters('tagEnvironment')]",
+        "Application": "[parameters('tagApplication')]",
+        "Project": "[parameters('tagProject')]",
+        "Owner": "[parameters('tagOwner')]",
+        "OwnerEmail": "[parameters('tagOwnerEmail')]"
+      },
+      "sku": {
+        "name": "[parameters('storageAccountSku')]"
+      },    
+      "properties": {
+        "supportsHttpsTrafficOnly": true
+      }
+    },
+    {
+      "type": "Microsoft.MachineLearning/workspaces",
+      "apiVersion": "[providers('Microsoft.MachineLearning', 'workspaces').apiVersions[0]]",
+      "name": "[parameters('workspaceName')]",
+      "location": "[parameters('location')]",
+      "tags": {
+        "Environment": "[parameters('tagEnvironment')]",
+        "Application": "[parameters('tagApplication')]",
+        "Project": "[parameters('tagProject')]",
+        "Owner": "[parameters('tagOwner')]",
+        "OwnerEmail": "[parameters('tagOwnerEmail')]"
+      },
+      "properties": {
+        "userStorageAccountId": "[variables('storageAccountResourceId')]",
+        "ownerEmail": "[parameters('ownerEmail')]"
+      },
+      "dependsOn": [
+        "[variables('storageAccountResourceId')]"
+      ]
+    },
+    {
+      "type": "Microsoft.MachineLearning/commitmentPlans",
+      "apiVersion": "[providers('Microsoft.MachineLearning', 'commitmentPlans').apiVersions[0]]",
+      "name": "[parameters('planName')]",
+      "location": "[parameters('location')]",
+      "tags": {
```

Comment:
Same as above, we might need to pass tags object as an input parameter

Reply:
tags now new format